syncd/Syncd.cpp

```diff
@@ -914,12 +1124,39 @@ sai_status_t Syncd::processBulkOid(
         SWSS_LOG_THROW("passing non object id to bulk oid obejct operation");
     }
 
-    // vendor SAI don't bulk API yet, so execute one by one
-
     std::vector<sai_status_t> statuses(objectIds.size());
 
     sai_status_t all = SAI_STATUS_SUCCESS;
 
+    if (m_commandLineOptions->m_enableSaiBulkSupport)
+    {
+        sai_bulk_op_error_mode_t mode = SAI_BULK_OP_ERROR_MODE_IGNORE_ERROR;
+
+        switch (api)
+        {
+            case SAI_COMMON_API_BULK_CREATE:
+                all = processBulkOidCreate(objectType, mode, objectIds, attributes, statuses);
+                break;
+
+            case SAI_COMMON_API_BULK_REMOVE:
+                all = processBulkOidRemove(objectType, mode, objectIds, statuses);
+                break;
+
+            default:
+                all = SAI_STATUS_NOT_SUPPORTED;
+                SWSS_LOG_ERROR("api %s is not supported in bulk mode", sai_serialize_common_api(api).c_str());
```

Comment:
missing break

Reply:
do we need break there? in any case we get out from switch